### PR TITLE
feat(cli): doctor command

### PR DESCRIPTION
**Pull request by jayzhang**
feat: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/25124858

### CODE REVIEW

File: packages/cli/src/resource/strings.json [hunk 1]
```diff
@@ -28,6 +28,23 @@
         "force": "Force upgrade the project to work with the latest version of Teams Toolkit."
       },
       "success": "Upgrade project successfully."
+    },
+    "doctor": {
+      "account": {
+        "SideLoadingDisabled": "Your Microsoft 365 tenant admin hasn't enabled sideloading permission for your account. You can't install your app to Teams!",
+        "NotSignIn": "No Microsoft 365 account login",
+        "SignInSuccess": "Microsoft 365 Account (%s) is logged in and sideloading permission enabled."
```
Comment (MuyangAmigo) [inline packages/cli/src/resource/strings.json:36]: NoSignIn: `You have not logged in to your Microsoft 365 account yet. Please use teamsapp auth login command to login to your Microsoft 365 account.`

File: packages/cli/src/resource/strings.json [hunk 2]
```diff
@@ -28,6 +28,23 @@
...
+      },
+      "node" : {
+        "NotFound": "Cannot find Node.js. Go to https://nodejs.org/about/releases/ to install LTS Node.js.",
+        "NotSupported": "Node.js (%s) is not the officially supported version (%s). Your project may continue to work but we recommend to install the supported version.",
```
Comment (MuyangAmigo) [inline packages/cli/src/resource/strings.json:40]: NotFound: `Cannot find Node.js. Node.js used for developing Teams apps with JavaScript or TypeScript using Teams Toolkit for Visual Studio Code. Visit https://nodejs.org/about/releases/ to install the LTS version.`

File: packages/cli/src/resource/strings.json [hunk 3]
```diff
@@ -28,6 +28,23 @@
...
+        "Success": "Node.js version (%s) is installed."
+      },
+      "dotnet" : {
+        "NotFound": "Cannot find .NET Core SDK. For the details why .NET SDK is needed, refer to %s.",
+        "NotSupported": ".NET Core SDK. (%s) is not the officially supported version (%s).",
```
Comment (MuyangAmigo) [inline packages/cli/src/resource/strings.json:45]: NotFound: `Cannot find .NET Core SDK. .NET Core SDK is used for developing Teams app in C# using Teams Toolkit for Visual Studio. Visit https://dotnet.microsoft.com/en-us/download to install the LTS version.`

File: packages/cli/src/resource/strings.json [hunk 4]
```diff
@@ -28,6 +28,23 @@
...
+        "Success": ".NET Core SDK. version (%s) is installed。"
+      }
```
Comment (MuyangAmigo) [inline packages/cli/src/resource/strings.json:47]: Extra `.` after SDK?

**Review by MuyangAmigo — DISMISSED** [review]

**Review by XiaofuHuang — DISMISSED** [review]

File: packages/cli/src/cmds/preview/constants.ts [hunk 5]
```diff
@@ -104,7 +104,7 @@ export const doctorResult = {
   SideLoadingDisabled:
     "Your Microsoft 365 tenant admin hasn't enabled sideloading permission for your account. You can't install your app to Teams!",
   NotSignIn: "No Microsoft 365 account login",
-  SignInSuccess: `Microsoft 365 Account (@account) is logged in and sideloading enabled`,
+  SignInSuccess: `Microsoft 365 Account (@account) is logged in and sideloading permission enabled`,
```
Comment (sffamily) [inline packages/cli/src/cmds/preview/constants.ts:107]: sideloading permission enabled -> sideloading permission is enabled

**Review by xzf0587 — DISMISSED** [review]

**Review by Alive-Fish — APPROVED** [review]

**Review by XiaofuHuang — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

**Review by kuojianlu — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]

**Review by sffamily — APPROVED** [review]

**Review by MuyangAmigo — APPROVED** [review]

**Pull request merged**